src/test/java/com/rocketden/main/api/RoomTests.java

```diff
@@ -445,6 +450,164 @@ public void changeRoomHostNewHostNotFound() throws Exception {
         assertEquals(ERROR.getResponse(), actual);
     }
 
+    @Test
+    public void updateRoomSettingsSuccess() throws Exception {
+        UserDto host = new UserDto();
+        host.setNickname("host");
+        UserDto user = new UserDto();
+        user.setNickname("test");
```

Comment:
The user is not used here, right? Could you remove these lines?

Reply:
It's only used in the call to `setUpRoomWithTwoUsers`. The second user isn't actually necessary for this test case tho, so lmk if you want and I can create another setup method that just creates a room with a single host, as that might make more sense

Reply:
Yeah I think that'd be helpful, if possible -- thanks!